src/js/main.js

```diff
@@ -89,6 +92,55 @@ Site.Paypal = {
 
 };
 
+Site.Player = {
+  init: function() {
+    var _this = this;
+
+    // Get the player container element
+    _this.$playeContainer = $('#player-container');
+
+    // Bind stuff
+    _this.bind();
+
+  },
+
+  bind: function() {
+    var _this = this;
+
+    // Init youtube whuen youtube api is ready
+    // TODO: subscribe to this event with jQuery
+    window.onYouTubePlayerAPIReady = _this.initYoutube.bind(_this);
+
+    // Listen for updatedyoutubelist
+    $(window).on('updatedyoutubelist', function(event, data) {
```

Comment:
This stuff about `list` is really not used on this feat yet but i intend to use it to track the list of videos.